tools/scripts/aducm.mk

```diff
@@ -283,4 +283,12 @@ clean_project:
  		-project $(PROJECT_NAME) \
  		-cleanOnly all
 
+PHONY += $(PLATFORM)_sdkclean
+$(PLATFORM)_sdkclean:
+	$(call print,[Delete] SDK created artefacts from $(BUILD_DIR))
```

Comment:
Could you please delete the "created" word ?
It generates this output and I find it very confusing whether something was created or deleted, even though by careful reading one may understand the proper meaning of it.
```
[iio_demo]$ make sdkclean
[09:52:42] [Delete] SDK created artefacts from /home/dari/Work/no-OS/projects/iio_demo/build/app/Debug
[09:52:42] [Delete] SDK created artefacts from /home/dari/Work/no-OS/projects/iio_demo/build/app/Release
```